### DISCUSSION TITLE
constant alerts about low battery

### DISCUSSION THREAD

**Discussion by nergdron** [nwg-piotr/nwg-shell#242] [Q&A]
hey all, I'm just starting to use nwg on hyprland, and while I mostly love it, I can't seem to figure out how to stop it from yelling at me that my battery is low in the notification area. I'm on a desktop, there's no battery on this system, but it keeps yelling that the battery is at 0%.

is this a bug in the default nwg-panel config? shouldn't it check to see if you even have a battery before it starts doing battery checks? how do I make it be quiet? :sweat_smile:

**Comment by nwg-piotr** [comment]
If on desktop, just uncheck the Battery check button. Setting 'Battery low notification [%]' to 0 will do the job as well. I did the latter in default configs on nwg-iso, but forgot to do the same in the nwg-shell repo. Will make a note to edit the files before the next release.

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/88f9b08c-ab7f-4083-82d7-a35d81762f7b)

**Comment by nwg-piotr** [comment]
0105137

**Comment by nergdron** [comment]
ahhh perfect, thanks. it should probably autodetect the presence of a battery before turning on those notifications, but being able to easily disable them is great.

**Comment by nergdron** [comment]
ok. unfortunately, I did this, but even after restarting my machine I keep getting the batter alerts. I've double checked, and in the panel prefs it shows as disabled, but they continue. so I'm confused for sure.

**Comment by nergdron** [comment]
I've also noticed a ton of messages in the systemd logs about backlight control not working, and when I toggle off the backlight control in the panel config and hit "apply/restart", it doesn't remove it and the messages continue. while not as annoying as the battery issue, it's still something I'd like to be able to disable.

**Comment by nergdron** [comment]
aha! this was user error, whoops! I was editing the config just called `config`, when nwg was configured to use the one after it, called `hyprland-0`. modules disabled correctly when I was using the correct config. ha!

**Reply by nwg-piotr** [reply]
I have something like 20 various configs, so I feel you pain. :rofl:
